bin/curriculum/remove_lesson_description_yml.rb

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+
+# removes the following lesson fields from scripts.en.yml:
+# - key
+# - description_student
+# - description_teacher
+
+require_relative '../../dashboard/config/environment'
+
+units_yml = File.expand_path("#{Rails.root}/config/locales/scripts.en.yml")
```

Comment:
I don't think its blocking but wondering if it would be safer to make sure we are on levelbuilder or local when running this script

Reply:
good call -- done!

Reply:
I reran it and confirmed that it still works in local development.